soccer/MainWindow.cpp

```diff
@@ -415,7 +429,9 @@ void MainWindow::updateViews() {
         _processor->refereeModule()->stage_time_left / 1000.0f, 'f', 2)));
 
     const char* blueName = _processor->refereeModule()->blue_info.name.c_str();
-    _ui.refBlueName->setText(strlen(blueName) == 0 ? "<Blue Team>" : blueName);
+    string blueFormatted = strlen(blueName) == 0 ? "Blue Team" : blueName;
```

Comment:
This is perfect, I didn't notice before, but without this, the styling wouldn't apply to teams set by the refbox. :+1: